Yoast/wordpress-seo
# Cleanup unused toggles code

## Context


* We want to deprecate unused code related to the old `Integrations` tab.

## Summary


This PR can be summarized in the following changelog entry:

* Deprecates unused code related to the old `Integrations` tab.

## Relevant technical choices:

* Even if it is true that the two toggles the code refers to are still used in multisite installations, they are rendered as `not available in multisites`.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

### Test the single site case
* Go to `Yoast SEO` -> `Integrations`
  * check that you can turn on and off the `Wincher` integrations without errors or warning
  * leave the integration on
* Edit a post
  * check the `Wincher` integration is in the metabox/sidebar
* Go to `Yoast SEO` -> `Integrations`
  * check that you can turn on and off the `WordProof` integrations without errors or warning
  * leave the integration on
* Edit your `Privacy Policy` page
  * In the `Advanced` section of the metabox/sidebar, check you can access the `Timestamp with WordProof` feature
* Go to `Yoast SEO` -> `Integrations`
  * turn off both features
*  Check you cannot access the two features from the metabox/sidebar

### Test the multisite case
* Go to `My Sites` -> `Network Admin` -> `Dashboard`
* Go to `Yoast SEO` -> `General` -> `Integrations card`
  * check there's no `WordProof integration` nor `Wincher integration` toggle
  * disable the `Semrush` integration and click  `Save changes`
    * check no errors/warnings are issued and you get the `Settings Updated.` banner
* Go to the dashboard of one of your sub-sites
  * go to `Yoast SEO` -> `Integrations`
  * check the `Semrush` card is disabled with a `Network disabled` badge in the heading
   
#### Relevant test scenarios


### Test instructions for QA when the code is in the RC



## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* N/A

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes [Cleanup code related to inactive multisite toggles#20780](https://github.com/Yoast/wordpress-seo/issues/20780)
